pkg/cue/process/handle.go

```diff
@@ -128,7 +136,8 @@ func NewContext(data ContextData) Context {
 		ctx:            data.Ctx,
 		baseHooks:      data.BaseHooks,
 		auxiliaryHooks: data.AuxiliaryHooks,
-		components:     data.Components,
```

Comment:
Why delete the components?

Reply:
sorry,my mistake,thanks for your reminder